lib/edge-cs.js

```diff
@@ -1,5 +1,9 @@
 var path = require('path');
 
 exports.getCompiler = function () {
-	return process.env.EDGE_CS_NATIVE || (process.env.EDGE_USE_CORECLR ? path.join(__dirname, 'edge-cs-coreclr', 'edge-cs-coreclr.dll') : path.join(__dirname, 'edge-cs.dll'));
+	return process.env.EDGE_CS_NATIVE || (process.env.EDGE_USE_CORECLR ? 'Edge.js.CSharp' : path.join(__dirname, 'edge-cs.dll'));
 };
+
+exports.getBootstrapDependencyManifest = function() {
+	return path.join(__dirname, 'bootstrap', 'bin', 'Release', 'netstandard1.5', 'bootstrap.deps.json');
+}
```

Comment:
This is now an extended contract of an Edge compiler? Could you describe it at https://github.com/tjanczuk/edge/wiki/Add-support-for-a-CLR-language?